zkevm-circuits/src/evm_circuit/execution/return_revert.rs

```diff
@@ -72,12 +82,6 @@ impl<F: Field> ExecutionGadget<F> for ReturnRevertGadget<F> {
 
         let is_success = cb.call_context(None, CallContextFieldTag::IsSuccess);
         cb.require_boolean("is_success is boolean", is_success.expr());
-        // cb.require_equal(
```

Comment:
This code should be reenabled.

Reply:
@lispc  find out even in success context,  op code can be `REVERT`,  `RETURN` can also cause failure.

Reply:
Ok but we are still missing something here. There is no difference between RETURN and REVERT in this code.

Reply:
see rely in next comment.